scripts/pre-commit

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+echo "*********************************************************"
+echo "Running git pre-commit hook. Running Spotless Apply... "
+echo "*********************************************************"
+
+[[ -s "$HOME/.sdkman/bin/sdkman-init.sh" ]] && source "$HOME/.sdkman/bin/sdkman-init.sh"
```

Comment:
Is this actually needed?

Reply:
Yes. IntelliJ in WSL with sdman runs a 'dumb' `/bin/sh` in which sdkman is not initialized (it is in bash/zsh, not in sh). Java is needed in this pre-commit hook because gradle/spotless is run